live-build/misc/ansible-roles/appliance-build.masking-development/tasks/main.yml

```diff
@@ -38,7 +38,6 @@
     update: no
   with_items:
     - { repo: 'https://gitlab.delphix.com/masking/dms-core-gate.git',
-        version: projects/dx4linux,
```

Comment:
I'm unable to comment directly on the line above, but since we're removing this version field from the items list, lets also remove it from line 37 where it's being used.

I think this would throw an error when running this role, since `item.version` is now undefined.

Reply:
Sure, done.